std_execution.bs

```diff
@@ -4448,6 +4503,7 @@ from the operation before the operation completes.
     </pre>
 
 1. `receiver_adaptor` is used to simplify the implementation of one receiver type in terms of another. It defines `tag_invoke` overloads that forward to named members if they exist, and to the adapted receiver otherwise.
+Any receiver queries (e.g., `get_stop_token`) defined in the base receiver are forwarded to the new receiver. 
```

Comment:
`get_stop_token` is a general query now, not specific to receivers. And it's only forwarded if `forwarding_receiver_query(CPO)` is a core constant expression that evaluates to `true`.